src/agentscope/service/file/common.py

```diff
@@ -201,3 +202,61 @@ def move_directory(
             status=ServiceExecStatus.ERROR,
             content=error_message,
         )
+
+
+def show_directory(directory_path: str) -> ServiceResponse:
+    """
+    Show/List the contents of a directory. i.e. ls -a
+
+    Args:
+        directory_path (`str`):
+            The path of the directory to show.
+
+    Returns:
+        `ServiceResponse`: The results contain a list of direcotry contents,
+        or an error message if any, including the error type.
+    """
+    if not os.path.exists(directory_path):
+        return ServiceResponse(
+            status=ServiceExecStatus.ERROR,
+            content="FileNotFoundError: The directory does not exist.",
+        )
+    if not os.path.isdir(directory_path):
+        return ServiceResponse(
+            status=ServiceExecStatus.ERROR,
+            content="FileNotFoundError: The path is not a directory",
+        )
+    try:
+        ls_result: List[str] = os.listdir(directory_path)
+        return ServiceResponse(
+            status=ServiceExecStatus.SUCCESS,
+            content=f"The listdir results are: {ls_result}",
```

Comment:
Return the list `ls_result` in content field directly.

Reply:
Done